ci-operator/config/CONFIGURATION.md

```diff
@@ -1 +1 @@
-See [CONFIGURATION.md](https://github.com/openshift/ci-tools/blob/master/CONFIGURATION.md).
+See [CONFIGURATION.md](https://github.com/openshift/ci-operator/blob/master/CONFIGURATION.md).
```

Comment:
Err that repo is archived. Please just link it to our docs: https://docs.ci.openshift.org/docs/architecture/ci-operator/